src/errors/publicErrors.ts

```diff
@@ -0,0 +1,55 @@
+import { constants as httpConstants } from 'node:http2'
```

Comment:
🟢 You don't have to address it on this PR, but wondering if could makes sense to also add `internalErrors`

Reply:
@kibertoad If you don't mind, I will move internal errors as well to `node-core`, want do you think?

Reply:
if there are some that we reuse across multiple services, sure!